README.rst

```diff
@@ -297,3 +297,24 @@ TLDR:
 
 Remember to activate your virtualenv whenever working on the repo, this is needed
 because pylint and mypy pre-commit hooks use the "system" python for now (because reasons).
+
+Pipeline public-key for JWT verification
+----------------------------------------
+
+Pipeline can pass a public-key whose private key is used to sign JWTs to
+access RM API with environment variable `$EXTERNAL_JWT_PUBKEY`. The public key
+should be base64-encoded to avoid issues with newlines and such when the key
+is passed through the system.
+
+Export a previously generated key to the container::
+
+    export RMAPI_EXTERNAL_JWT_PUBKEY=$(cat ~/p/jwt-test/private_key.key| base64)
```

Comment:
Should we add _B64 to the variable name ? Explicit is better than implicit.

Reply:
sure, can do